### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by FynnBe**
Update docs to match paper draft

### CODE REVIEW

File: docs/contribute_models/README.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ Ready to follow the [Tutorial](/contribute_models/tutorials.md)?
 
 ## Contributing other resource types
 
-To contribute a notebook, application or dataset, please use the generic [Resource Description File Format](https://github.com/bioimage-io/spec-bioimage-io/blob/main/README.md).
+To contribute a notebook, application or dataset, please use the generic [Resource Description File Format](https://github.com/bioimage-io/spec-bioimage-io/blob/main/README.md#resource-description-file-specification).
```
Comment (FynnBe) [inline docs/contribute_models/README.md]: @oeway can we link markdown section like this in the imjoy docs?

Comment (FynnBe) [reply docs/contribute_models/README.md]: oh, well as links of course this is possible.... I was thinking of adding them as their own doc pages first.. but let's just leave it as link now and then this is totally fine..

File: docs/contribute_models/README.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ Model contribution means that you will upload a fully-documented trained model t
 - It is assigned a unique DOI identifier and a License.
 - The model is public and can be used by anyone under the chiosen licensing conditions.
 
-The model package contains the trained weights together with the architecture, example inputs and outputs, and the configuration specification file that describes your model technically in such a way that at least one of the consumer software can load and run the model. All this information is embedded in a specific file called [`Model Resource Description File` (RDF)](). 
+The model package contains the trained weights together with the architecture, example inputs and outputs, and the configuration specification file that describes your model technically in such a way that at least one of the consumer software can load and run the model. All this information is embedded in a specific file called [`Model Resource Description File` (RDF)](https://github.com/bioimage-io/spec-bioimage-io/blob/main/README.md#model-resource-description-file-specification).
```
Comment (oeway) [inline docs/contribute_models/README.md]: We can add a link to the docs so the users won't need to leave the docs, I will add that later.

Comment (FynnBe) [reply docs/contribute_models/README.md]: yes, that is what I had in mind..  can we link markdown sections 'in-place' like that, too?

File: docs/community_partners/partner_collection.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  # the available resource types in your collection
+  resource_types:
+    - model
+    - application
+    - notebook
+    - dataset
+
+  # the default resource type you want to set, set to `all` if you want to show all your items by default
+  default_type: all
```
Comment (FynnBe) [inline docs/community_partners/partner_collection.md:49]: @oeway are all of these config fields still used by the website?

File: docs/contribute_models/README.md (hunk 2 above)
Comment (oeway) [reply docs/contribute_models/README.md]: I just checked, we acutally have it already mounted to the docs under: https://bioimage.io/docs/#/bioimageio_spec?id=model-resource-description-file-specification

This is where we can add these alias: https://github.com/bioimage-io/bioimage.io/blob/da5b79c06b9a0dc6e403454c1a3240e709f1261c/docs/index.html#L49-L53

File: docs/community_partners/partner_collection.md (hunk 3 above)
Comment (oeway) [reply docs/community_partners/partner_collection.md:49]: Yes, they do

Comment (oeway) [reply docs/community_partners/partner_collection.md:49]: Yes!

File: docs/contribute_models/tutorials.md [hunk 4]
```diff
@@ -1,18 +1,16 @@
 
 # Tutorial for contributing models
-**Note: only PyTorch and TensorFlow models are supported at the moment. We are working on alternative model formats such as ONNX.**
 
-1. Check that the (1) that your model is supported by the BioImage Model Zoo and (2) it is compatible with at least one of the consumer software. Check it [here](https://github.com/bioimage-io/spec-bioimage-io/blob/master/supported_formats_and_operations.md).
+1. Create a model resource description file (RDF) according to the [BioImage.IO Model Resource Description File Specification](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/model_spec_latest.md).
```
Comment (oeway) [inline docs/contribute_models/tutorials.md]: Here, the alias URL in the docs is: https://bioimage.io/docs/#/bioimageio_model_spec

File: docs/contribute_models/tutorials.md [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
+   - Each field on the file is either mandatory or optional. You can use [our template](https://github.com/bioimage-io/bioimage-io-models/pull/55/files#diff-f6c64be5b9d764d0964654908b2ed4495fccc7624e58e9360bfdc6cef169edbe) to fill in the required information. 
+   - [Here is an example](https://github.com/bioimage-io/pytorch-bioimage-io/blob/master/specs/models/unet2d_nuclei_broad/UNet2DNucleiBroad.model.yaml) of a valid model RDF. In the Bioimage Model Zoo web page you will also find different examples. 
+   - You can validate your model RDF with the `bioimageio validate` command as described [here](https://github.com/bioimage-io/spec-bioimage-io#validate).
```
Comment (oeway) [inline docs/contribute_models/tutorials.md]: https://bioimage.io/docs/#/bioimageio_spec?id=validate

Comment (FynnBe) [reply docs/contribute_models/tutorials.md]: just noticed a bug there in the sidebar. in the Changelogs section
image

Comment (oeway) [reply docs/contribute_models/tutorials.md]: Merge the PR in the spec repo should fix it.

Comment (oeway) [reply docs/contribute_models/tutorials.md]: This one: https://github.com/bioimage-io/spec-bioimage-io/pull/457

**Pull request merged**